cumulus/parachains/runtimes/glutton/glutton-kusama/src/lib.rs

```diff
@@ -224,7 +224,10 @@ impl cumulus_pallet_aura_ext::Config for Runtime {}
 impl pallet_timestamp::Config for Runtime {
 	type Moment = u64;
 	type OnTimestampSet = Aura;
+	#[cfg(feature = "experimental")]
 	type MinimumPeriod = ConstU64<0>;
+	#[cfg(not(feature = "experimental"))]
+	type MinimumPeriod = ConstU64<{ SLOT_DURATION }>;
```

Comment:
I didn't divide it by two because `SLOT_DURATION` here is defined as 6000ms already (as opposed to the duration used in the other runtimes, which is 12000ms).

I'm no expert on aura, should this duration be 3000ms as you suggest?

Reply:
Actually, I tested this again and I run into another runtime panic if I leave `MinimumPeriod  = 6000ms`->  `slot number mismatch`

Reply:
`MINIMUM_PERIOD` is the minimum period that should have passed between blocks. And 3 seconds is here correct. This means that collators at least need to wait 3 seconds before building a new block.